stable/datadog/values.yaml

```diff
@@ -141,6 +141,9 @@ datadog:
   ##
   # nonLocalTraffic: true
 
+  ## Enable docker socket volume mounting
+  useDockerSocketVolume: True
```

Comment:
not Python :)
use true

Reply:
I'll make this change if that's your convention, but note that this is perfectly valid YAML. Booleans [are matched with](http://yaml.org/type/bool.html):

```
 y|Y|yes|Yes|YES|n|N|no|No|NO
|true|True|TRUE|false|False|FALSE
|on|On|ON|off|Off|OFF
```

YAML is fun!